features/step_definitions/viewing_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given(/^I am on the root page$/) do
+  visit '/'
+end
+
+Given(/^there is a post to view$/) do
+  @otherUser = FactoryGirl.create(:user,
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)